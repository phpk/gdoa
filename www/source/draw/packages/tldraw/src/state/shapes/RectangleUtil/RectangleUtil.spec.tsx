import { Rectangle } from '..'

describe('Rectangle shape', () => {
  it('Creates a shape', () => {
    expect(Rectangle.create({ id: 'rectangle' })).toMatchSnapshot('rectangle')
  })
})
