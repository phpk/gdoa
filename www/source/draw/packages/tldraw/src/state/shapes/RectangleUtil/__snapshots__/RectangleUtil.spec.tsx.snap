// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rectangle shape Creates a shape: rectangle 1`] = `
Object {
  "childIndex": 1,
  "id": "rectangle",
  "name": "Rectangle",
  "parentId": "page",
  "point": Array [
    0,
    0,
  ],
  "rotation": 0,
  "size": Array [
    1,
    1,
  ],
  "style": Object {
    "color": "black",
    "dash": "draw",
    "isFilled": false,
    "scale": 1,
    "size": "small",
  },
  "type": "rectangle",
}
`;
